spec/gogo/x/gov/codec.proto

```diff
@@ -224,7 +232,8 @@ message TallyMsg {
 // with a reference to the electorate that approved it
 message TextResolutionMsg {
   weave.Metadata metadata = 1;
-  string resolution = 2;
+  bytes proposal_id = 2 [(gogoproto.customname) = "ProposalID"];
```

Comment:
The proposal id is also auto-assigned. The point is we need to pass in some data post-election-tally to the executor so it knows why it is being executed.

I would add something to the context we pass to the Executor. And add a public accessor to x/gov to read that info (so any handler can choose to be gov-aware). Passing proposal_id (or the Proposal object itself) should be enough, as they can query the rest of the context from that object.